examples/parameters.rs

```diff
@@ -0,0 +1,86 @@
+//! FMOD local parameter example usage:
+//! Press H and L to change the amount of traffic you can hear.
+//! Press E and M to change the time to Evening and Morning respectively.
+//! Note that because `ignore_seek_speed` is `false` in these examples,
+//! it could take a few seconds before you can hear the changes in these sound effects.
+
+use bevy::prelude::*;
```

Comment:
Wildcard import

Reply:
I've been thinking about this, for code inside our library I understand your reasoning, but for the examples I think we ought to use what is the most common way for Bevy users to import stuff. That way we create realistic and clean examples.

It's even the recommended way in the Bevy Book and in this unofficial document: https://github.com/tbillington/bevy_best_practices#project-structuring

Afaik `prelude::*` is more of a curated wildcard as well.
Let me know what you think

Reply:
Hmm... If it's encouraged this heavily in the community, I'm fine with doing it in the examples. Can you adjust the imports in the examples then?

Reply:
Will do